types/roslib/index.d.ts

```diff
@@ -282,7 +282,10 @@ declare namespace ROSLIB {
         delete(callback: (response: any) => void): void;
     }
 
-    export class Service {
+    export class Service <
+        TServiceRequest = any,
```

Comment:
Should the default be `ServiceRequest` to match the old behavior?

Reply:
The issue is that https://github.com/DefinitelyTyped/DefinitelyTyped/pull/55066/files#diff-726d2dc6f8cec1d3206d29c4e5f5207d9b7ca4e967c7295abca636e6976ddb1cL324 is typed as `any` rather than `ServiceRequest`. Defaulting the generic to `any` keeps it backwards compatible whereas doing `TServiceRequest = ServiceRequest` causes the tests to fail on line [87](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/725f962de94a26a880a34a0f6e0d277b7512e68a/types/roslib/roslib-tests.ts#L87):
```
ERROR: 87:20  expect  TypeScript@4.4 compile error: 
Property 'a' does not exist on type 'ServiceRequest'.
ERROR: 87:28  expect  TypeScript@4.4 compile error: 
Property 'b' does not exist on type 'ServiceRequest'.
```

Reply:
You could try `TServiceRequest extends ServiceRequest = any` but it looks like `ServiceRequest` has no properties, so it doesn't really matter.